makeDarwinImage/default.nix

```diff
@@ -5,7 +5,7 @@
 let
   diskSize = if diskSizeBytes < 40000000000 then throw "diskSizeBytes ${toString diskSizeBytes} too small for macOS" else diskSizeBytes;
 
-  installAssistant-fetched = import <nix/fetchurl.nix> {
+  installAssistant-fetched = fetchurl {
```

Comment:
Feel free to drop this, it's here just for my convenience. Without it I cannot fetch the image, don't know why but it's not related to the PR's goal